src/App.js

```diff
@@ -88,7 +90,8 @@ export default class App extends Component {
     const AppContainer = createAppContainer(AppNavigator);
 
     return (
-      <Provider store={store}>
+    <Provider store={store}>
```

Comment:
Not a bit deal, but there are some whitespace issues here that would probably be fixed with our linter.